src/BloomExe/web/ReadersHandler.cs

```diff
@@ -255,6 +258,13 @@ private static void MakeLetterAndWordList(string jsonSettings, string allWords)
 			PathUtilities.OpenFileInApplication(fileName);
 		}
 
+		private static string GetLocalizedString(string key, string defaultText)
+		{
+			return LocalizationManager.UILanguageId == "en"
+				? LocalizationManager.GetDynamicString("Bloom", key, defaultText)
```

Comment:
I basically duplicated what is being done in the RuntimeInformationInjector. It looks like we only want to collect new strings in the English tmx file.